doc/classes/AABB.xml

```diff
@@ -33,211 +33,334 @@
 			<param index="0" name="position" type="Vector3" />
 			<param index="1" name="size" type="Vector3" />
 			<description>
-				Constructs an [AABB] from a position and size.
+				Constructs an [AABB] by [param position] and [param size].
 			</description>
 		</constructor>
 	</constructors>
 	<methods>
 		<method name="abs" qualifiers="const">
 			<return type="AABB" />
 			<description>
-				Returns an AABB with equivalent position and size, modified so that the most-negative corner is the origin and the size is positive.
+				Returns an [AABB] equivalent to this bounding box, with its width, height, and depth modified to be non-negative values.
+				[codeblocks]
+				[gdscript]
+				var box = AABB(Vector3(5, 0, 5), Vector3(-20, -10, -5))
+				var absolute = box.abs()
+				print(absolute.position) # Prints (-15, -10, 0)
+				print(absolute.size)     # Prints (20, 10, 5)
+				[/gdscript]
+				[csharp]
+				var box = new Aabb(new Vector3(5, 0, 5), new Vector3(-20, -10, -5));
+				var absolute = box.Abs();
+				GD.Print(absolute.Position); // Prints (-15, -10, 0)
+				GD.Print(absolute.Size);     // Prints (20, 10, 5)
+				[/csharp]
+				[/codeblocks]
+				[b]Note:[/b] It's recommended to use this method when [member size] is negative, as most other methods in Godot assume that the [member size]'s components are greater than [code]0[/code].
 			</description>
 		</method>
 		<method name="encloses" qualifiers="const">
 			<return type="bool" />
 			<param index="0" name="with" type="AABB" />
 			<description>
-				Returns [code]true[/code] if this [AABB] completely encloses another one.
+				Returns [code]true[/code] if this bounding box [i]completely[/i] encloses the [param with] box. The edges of both boxes are included.
+				[codeblocks]
+				[gdscript]
+				var a = AABB(Vector3(0, 0, 0), Vector3(4, 4, 4))
+				var b = AABB(Vector3(1, 1, 1), Vector3(3, 3, 3))
+				var c = AABB(Vector3(2, 2, 2), Vector3(8, 8, 8))
+
+				print(a.encloses(a)) # Prints true
+				print(a.encloses(b)) # Prints true
+				print(a.encloses(c)) # Prints false
+				[/gdscript]
+				[csharp]
+				var a = new Aabb(new Vector3(0, 0, 0), new Vector3(4, 4, 4));
+				var b = new Aabb(new Vector3(1, 1, 1), new Vector3(3, 3, 3));
+				var c = new Aabb(new Vector3(2, 2, 2), new Vector3(8, 8, 8));
+
+				GD.Print(a.encloses(a)); // Prints true
+				GD.Print(a.encloses(b)); // Prints true
+				GD.Print(a.encloses(c)); // Prints false
```

Comment:
If the print comment is meant to be accurate to what actually gets printed then it needs to be capitalized.

```suggestion
				GD.Print(a.Encloses(a)); // Prints False
				GD.Print(a.Encloses(b)); // Prints True
				GD.Print(a.Encloses(c)); // Prints False
```

Reply:
It is meant to, thank you. So C#'s `GD.Print()` and GDScript's `print()` are ever so slightly different this way?

Reply:
It should be `True, True, False`, you probably aren't running the latest version

Reply:
It's okay, I did understand the assignment

Reply:
(didn't notice your comment before mine 🙂)

Reply:
`GD.Print` uses the type's `ToString` method and the `System.Boolean.ToString` method capitalizes the output like that.

@AThousandShips Oh I see. The changes from https://github.com/godotengine/godot/pull/87118 need to also be applied to C# in:

https://github.com/godotengine/godot/blob/7a90c56c00e7d63199bd3abe409622c3c24e0ec5/modules/mono/glue/GodotSharp/GodotSharp/Core/Aabb.cs#L93-L106

Reply:
Oh then I'll fix that, thank you!

Reply:
Done:
* https://github.com/godotengine/godot/pull/87264

Not familiar with C# in 3.x so will have to research how to fix that on that version as a separate PR